src/cli.ts

```diff
@@ -165,3 +181,12 @@ function wrapAction(
     }
   };
 }
+
+/**
+ * Array flags, when not passed, are either `undefined` or `[undefined]`. This function filters out
+ * the
+ */
+function getArrayFromFlags<T>(flags: any, name: string): T[] | undefined {
+  const array = [flags[name]].flat() as Array<T | undefined>;
+  return array.filter((item) => item != null) as T[];
+}
```

Comment:
@aklinker1 The result will be empty array at least, so it caused `userConfig` of `mergeInlineConfig(inlineConfig, userConfig)` has no effect.

After released v1.4.2, I have to patch wxt temporarily.